facturacion_mexico/facturacion_fiscal/services/ffm_reconciliation.py

```
@@ -221,6 +236,15 @@ def _reconcile_ffm(ffm_name: str) -> dict:
 				ffm.name, fiscal_status, sync_status=sync_status, cancellation_date=_cdate
 			)
 
+			# Sustitución motivo 01: si el CFDI quedó CANCELADO y es ORIGEN de una sustitución (existe
+			# un CFDI que lo relaciona vía ffm_substitution_source_uuid) con documentos aún en
+			# docstatus=1, completar la cancelación DOCUMENTAL pendiente (idempotente, sin tocar el PAC).
+			if fiscal_status == FiscalStates.CANCELADO:
+				try:
+					_reconcile_substitution_documental(ffm)
+				except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)